src/screens/NetworkSettings/NetworkSettingsFlowStack.js

```diff
@@ -7,12 +8,13 @@ export const NetworkSettingsFlowStack = ({ navigation }) => {
   const FlowStack = createStackNavigator();
   return (
     <FlowStack.Navigator
-      initialRouteName={NetworkSettingsDislaimerNav}
+      initialRouteName={NetworkPreSettingsNav}
```

Comment:
Why don't we start with the disclaimer screen? Even changing the server to testnet should show a warning, so I think we could always show it.

Reply:
Oh. Sorry, I let my development process interfere here. But the definite initialRoute is set in PR (11).

Reply:
Why are you adding this change in this PR? I think this shouldn't be committed because it might lead to mistakes.